scripts/get-dep.sh

```diff
@@ -57,8 +57,8 @@ function get_json()
 
 function get_catch()
 {
-	GIT_REPO="https://github.com/philsquared/Catch.git"
-	GIT_TAG="v2.11.0"
+	GIT_REPO="https://github.com/catchorg/Catch2.git"
```

Comment:
`https://github.com/philsquared/Catch` is renamed. At the moment, the catch repository URL is `https://github.com/catchorg/Catch2`.